### PR TITLE
Import Terraform main.tf and define Windows test app.

### PR DESCRIPTION
**Pull request by jeschkies**
## High-level description

This imports a parameterized `main.tf` that can be used for local and CI testing.

Furthermore the Windows test app is adapted to it launches a simple ASP app with
health checks.

Outstanding:
- [x] At README with example usage.
- [x] dcos-test-utils changes in https://github.com/dcos/dcos-test-utils/pull/103

### CODE REVIEW

File: packages/dcos-integration-test/README.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ The ACS token is valid only once.
 Run the simple test with
 
 ```bash
+export DCOS_ACS_TOKEN="$(dcos config show core.dcos_acs_token)"
```
Comment (jeschkies) [inline packages/dcos-integration-test/README.md]: This requires the fix https://github.com/dcos/dcos-test-utils/pull/103

**Comment by jeschkies** [comment]
@mesosphere-mergebot label Ready for Review

**Comment by jeschkies** [comment]
@mesosphere-mergebot changelog-not-required This is an internal change

**Comment by jeschkies** [comment]
@fatz, could you review as well.

File: test_util/main.tf [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  version = "~> 0.2.0"
+
+  cluster_name           = "${local.cluster_name}"
+  hostname_format        = "%[3]s-winagent%[1]d-%[2]s"
+  aws_subnet_ids         = ["${module.dcos.infrastructure.vpc.subnet_ids}"]
+  aws_security_group_ids = ["${module.dcos.infrastructure.security_groups.internal}", "${module.dcos.infrastructure.security_groups.admin}"]
+  aws_key_name           = "${module.dcos.infrastructure.aws_key_name}"
+
+  # provide the number of windows agents that should be provisioned.
+  num = "1"
```
Comment (fatz) [inline test_util/main.tf]: maybe this could be a variable? `num_winagents`?

File: test_util/main.tf [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  num_masters        = "1"
+  num_private_agents = "1"
+  num_public_agents  = "1"
+
+  dcos_instance_os        = "centos_7.5"
+  bootstrap_instance_type = "m4.xlarge"
+
+  dcos_variant              = "${var.variant}"
+  dcos_version              = "2.1.0-beta1"
+  dcos_license_key_contents = "${file("~/license.txt")}"
```
Comment (janisz) [inline test_util/main.tf]: This should be a [variable](https://learn.hashicorp.com/terraform/getting-started/variables.html)

Comment (janisz) [reply test_util/main.tf]: but we can fix it later

**Comment by jeschkies** [comment]
@mesosphere-mergebot changelog-not-required simple internal change

**Comment by jeschkies** [comment]
@mesosphere-mergebot sync

**Comment by jeschkies** [comment]
This war merge with e5548027f3b70a56e9e151284e17f4dbe14cc55e

**Pull request closed without merging**